recipes/osmesa/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+./configure --prefix=$PREFIX --enable-osmesa --disable-gallium --disable-egl --with-drivers=osmesa
+
+make install
```

Comment:
TBH I have no idea, I'm writing this on a Windows machine and the docs are incredibly unclear. :/ There's word of an OpenGL conformance test but it's closed source, so that's not an option.